reggie/ingestion/download.py

```diff
@@ -170,6 +170,7 @@ def __init__(self, config_file=CONFIG_OHIO_FILE, force_date=None,
         self.state = config["state"]
         self.obj_will_download = False
         self.meta = None
+#        self.dataframe = None
```

Comment:
maybe clean up?

Reply:
tinkering w/ dataframe output, will clean off my master branch (right now read_csv is called twice, was testing what would happen if we saved the preprocess dataframe as an attribute to return instead, but don't know how that would affect memory management)